src/main/host/syscall_handler.c

```diff
@@ -165,6 +165,12 @@ static void _syscallhandler_post_syscall(SysCallHandler* sys, long number,
         scr = syscallhandler_##s(sys, args);                                   \
         _syscallhandler_post_syscall(sys, args->number, #s, &scr);             \
         break
+#define HANDLE_NEW(s)                                                              \
```

Comment:
Throughout, is there a more descriptive tag we can use than `new`? maybe `rust`? e.g. `HANDLE_RUST`?

Reply:
It is now `HANDLE_RUST`.